src/cli.rs

```diff
@@ -65,6 +66,9 @@ pub fn parse(target_list: &TargetList) -> Args {
     let docker_in_docker = env::var("CROSS_DOCKER_IN_DOCKER")
         .map(|s| bool::from_str(&s).unwrap_or_default())
         .unwrap_or_default();
+    let force_image = env::var("CROSS_FORCE_IMAGE")
+        .map(|s| bool::from_str(&s).unwrap_or_default())
```

Comment:
This can now use `bool_from_envvar`, which should fix the issue mentioned in the README.